NAMESPACE

```diff
@@ -2,6 +2,11 @@
 
 S3method(`[`,scores)
 S3method(as_forecast,default)
+S3method(as_point,default)
+S3method(as_point,forecast_quantile)
+S3method(as_point,forecast_sample)
+S3method(as_quantile,default)
+S3method(as_quantile,forecast_sample)
```

Comment:
Only minor comment is that this feels as if there is a new `point`, or `quantile` class. I believe it would make sense, and be slightly easier to wrap my head around as a potential user, if we re-used the existing classes and terminology. I.e., `forecast_point` and `forecast_quantile`.

This would lead to longer names but that's probably still ok :thinking:

Reply:
Yeah I think if we are going with a abstract class as in #373 we should rename.